src/components/TalkToUs/LeaveMessage/index.js

```diff
@@ -141,7 +144,12 @@ const Callback = () => {
           </p>
         ) : null}
         <Button type="submit" css={submitButton} disabled={disabled}>
-          SUBMIT
+          SUBMIT{" "}
+          <ClipLoader
+            size={14}
```

Comment:
This component supports `css` props, right? So you don't need to use `useTheme`

Reply:
btw this component `css` props is not work, so i use `useTheme` instead.